add-ons/aggregations-view.js

```diff
@@ -104,8 +104,12 @@ AggregationsView.prototype.setAggregateGroups = function(aggregations, groups) {
     if (aggregated) {
         dataSource.setAggregateGroups(aggregations, groups);
         behavior.reindex(); // rows have changed
+        behavior.setTopTotals(behavior.dataModel.dataSource.getGrandTotals());
```

Comment:
These lines should not be necessary. See lines [16](https://github.com/openfin/fin-hypergrid/blob/develop/src/dataModels/SummarySubgrid.js#L16) and [35](https://github.com/openfin/fin-hypergrid/blob/develop/src/dataModels/SummarySubgrid.js#L35) of SummarySubgrid.js.